script/tool/lib/src/common/package_command.dart

```diff
@@ -545,6 +572,32 @@ abstract class PackageCommand extends Command<void> {
     return packages;
   }
 
+  String? _getCurrentDirectoryPackageName() {
+    // Ensure that the current directory is within the packages directory.
+    final Directory absolutePackagesDir = packagesDir.absolute;
+    Directory currentDir = packagesDir.fileSystem.currentDirectory.absolute;
+    if (!currentDir.path.startsWith(absolutePackagesDir.path) ||
+        currentDir.path == packagesDir.path) {
+      return null;
+    }
+    // If the current directory is a direct subdirectory of the packages
```

Comment:
sub-directory

Reply:
"subdirectory" is a word (and according to https://www.merriam-webster.com/dictionary/subdirectory is the more common form).

Reply:
hmm, can't argue with that. Seems a little strange to me.

Reply:
I was trying to think of other examples, and it seems like sub is mostly not hyphenated. I'm not sure if there's a rule, or it's one of those things that gets dropped in anything where it's used frequently.

Reply:
subbasement, subfloor... I'm just wrong I guess.